### DISCUSSION TITLE
除了阿里自身淘系产品用XQUIC，是否还有其它公司使用XQUIC成功落地产品？

### DISCUSSION THREAD

**Discussion by nilins** [alibaba/xquic#406] [General]
初步了解XQUIC 后。
1、除了阿里自身淘系产品用XQUIC，是否还有其它公司使用XQUIC成功落地产品？
2、有使用公司的名单么？ 
3、未来是否支持FEC的计划？ 
4、multipath quic是包复制发包，还是一个流的包 可以同时在多路径上发数据

**Comment by Kulsk** [comment]
1/2：有一些其他公司在使用，但涉及到第三方公司的信息安全，不方便透露；
3：FEC正在开发中；
4：不是特别明白你的意思；

**Reply by nilins** [reply]
感谢你的回复。

对于多路径发包，有两个场景。
比如应用要发4个数据包 A、B、C、D。有两条路径都是available path。 
场景1： 路径1 发 A、B、C、D， 路径2 发 A、B、C、D 。收到包，后到的相同数据包丢弃，再重组成 ABCD
场景2：路径1发 A C ，路径2 发B D。收到包后重组成 ABCD。

我的问题是草案 的建议， 是不是场景1 和场景2 都能支持， 由应用决策？
